src/AccessibilityInsights.Extensions.AzureDevOpsTests/FileIssue/FileIssueHelpersTests.cs

```diff
@@ -0,0 +1,69 @@
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using AccessibilityInsights.Extensions.AzureDevOps;
+using AccessibilityInsights.Extensions.AzureDevOps.Fakes;
+using AccessibilityInsights.Extensions.AzureDevOps.FileIssue;
+using AccessibilityInsights.Extensions.Interfaces.IssueReporting;
+using Microsoft.QualityTools.Testing.Fakes;
+using Microsoft.VisualStudio.TestTools.UnitTesting;
+using System;
+using System.Collections.Generic;
+using System.Threading.Tasks;
+
+namespace AccessibilityInsights.Extensions.AzureDevOpsTests.FileIssue
+{
+    [TestClass]
+    public class FileIssueHelpersTests
+    {
+        static readonly Uri FAKE_SERVER_URL = new Uri("https://myaccount.visualstudio.com/");
+
+        [TestMethod]
+        public void FileNewIssue_IsNotEnabled_ReturnsPlaceholder()
+        {
+            using (ShimsContext.Create())
+            {
+                ShimAzureDevOpsIntegration.AllInstances.ConnectedToAzureDevOpsGet = (_) => false;
+                var issueInfo = new IssueInformation();
+                var connInfo = new ConnectionInfo();
+                var output = FileIssueHelpers.FileNewIssue(issueInfo,
+                    connInfo, false, 0, (_) => { });
+
+                Assert.IsNull(output.issueId);
+                Assert.IsNotNull(output.newIssueId);
+                Assert.IsTrue(string.IsNullOrEmpty(output.newIssueId));
+            }
+        }
+
+
```

Comment:
Can we remove one of the extra newlines between these 2 tests?

Reply:
Only for you, Dave